.github/workflows/basic.yml

```diff
@@ -129,7 +129,7 @@ jobs:
       working-directory: ${{env.BUILD_DIR}}
       run: >
         ${{ matrix.compiler.cxx == 'icpx' && '. /opt/intel/oneapi/setvars.sh &&' || ''}} 
-        ctest --output-on-failure --test-dir test
+        ctest --output-on-failure --test-dir test -LE "fuzz-long"
```

Comment:
As an user i do not want to each time remember to disable this test - i just want type "ctest" - if this test has to be disabled "in normal use".,please create cmake flag to enable this test.

Reply:
I agree, we could have a generic `UMF_TESTS_ENABLE_LONG`

Reply:
Done